build_tools/python/e2e_test_framework/unique_ids.py

```diff
@@ -85,6 +85,15 @@ def hash_composite_id(keys: Sequence[str]) -> str:
 #    PyTorch.
 MODEL_CLIP_TEXT_SEQLEN64_FP32_TORCH = "9a9515c7-cb68-4c34-b1d2-0e8c0a3620b8"
 MODEL_UNET_2D_FP32_TORCH = "340553d1-e6fe-41b6-b2c7-687c74ccec56"
+MODEL_BERT_LARGE_1x384_FP32_TORCH = "cbc5e400-7c93-4844-aca8-bce8f1bf9948-batch-1"
```

Comment:
nit: "x" in the middle should be capitalized?

Reply:
Updated to "X"